mcp/motors/actuators.c

```diff
@@ -799,36 +799,23 @@ static int GetShutterData(struct ezbus* pBus, char who, int* lims, int* pos)
     return retval;
 }
 
+
 /**
- * @brief Switch cases to call shutter move functions based on CommandData
+ * @brief Decides the shutter actuator action to take based on the goal.
+ * @param[in,out] shutter_state (uint32_t*) pointer to shutter_data.state, since it may
+ * need to be updated
+ * @param[in,out] shutter_goal (uint32_t*) pointer to CommandData.actbus.shutter_goal,
+ * since it may need to be updated
+ * @return action (uint32_t)
```

Comment:
Doxygen will include function signature in the documentation it generates (see below)

![Screenshot 2023-07-10 at 15-35-48 MCP motors_actuators c File Reference](https://github.com/tim-balloon/TIMflight/assets/1486086/28e149c5-f0c6-4fdf-9128-6cf5b9382b20)

As a result including them in the function description seem a little bit redundant. But it's just my opinion, not a fundamental objection. :wink:

Reply:
Should I just remove `(uint32_t*)`, etc? The signature above has the names and types and stuff, but I like being able to note whether the param are inputs, outputs, or both.

Reply:
I was just talking about `(uint32_t*)`. `[in, out]` is totally fine with me.